### ISSUE TITLE
[bug]: Accessibility/A11y: Missing icons in Breadcrumb in HCM mode of Windows

### ISSUE THREAD

**Issue by aedu-hostettler** [swisspost/design-system#3443]
**Describe the bug**
When High Contrast Mode (HCM) in Windows is activated, in non-Firefox Browsers, the Icons are missing between the Breadcrumb links/entries. Concrete the Home(House) Icon most left, and the "Chevron right" Icons between each entry.

**To Reproduce**

1. Go to https://design-system.post.ch/?path=/docs/b93adeff-9bb8-4f23-a81d-96452a94fd9b--docs
2. See the first example (as in provided Screenshot with HCM turned on and off for comparison. (Missing icons marked red in HCM version.)

**Expected behavior**
The Icons should also show up in HCM mode, as they are visible in normal/non-HCM mode.

**Screenshots**
![breadcrumbs](https://github.com/user-attachments/assets/63abfd56-68ad-40ba-96c9-000f4af02d00)

**Desktop (please complete the following information):**

- OS: Windows 10/11 (happens on both)
- Browser: Chrome, Edge (not tested with Safari or Opera). (Icons are correctly displayed with Firefox Browser)
- Version: current versions at 21.8.2024.

**Smartphone (please complete the following information):**
not testet on mobile-browsers

**Additional context**
Seems to happen with older and current Version of Internet-Header (also Intranet-Header?) 1.14.4/8.1.0 (not sure if belongs to a common Breadcrumb-Component in general, or specific/individual Inter/Intranet-Header components.), and this would also only apply to the concrete versions, if the link above (examples on https://design-system.post.ch/) are based on the latest versions of your packages.

**Comment by gfellerph** [comment]
Thanks @aedu-hostettler for submitting this issue. We're having a look at it as soon as possible.
